docs/nuvlaedge/installation/requirements.md

```diff
@@ -11,40 +11,61 @@ redirect_from:
+To ensure a smooth and fully functional installation of the NuvlaEdge (**NE**),
+please make sure you comply with the following system requirements.
 
 
 ## OS Requirements
 
-The NuvlaEdge software is compliant with any OS that officially and fully supports Docker. We recommend one of the following:
+The NuvlaEdge software is compliant with any OS that officially and fully
+supports Docker or Kubernetes. We recommend one of the following:
+
 - Ubuntu
 - CentOS
 - Debian (and Debian variants like Raspbian and Raspberry Pi OS)
 
-Other Docker-compliant Linux distributions should also work, but the NuvlaEdge functionality will be subject to the level of Docker support the OS provides. 
+Other Docker and Kubernetes compliant Linux distributions should also work, but
+the NuvlaEdge functionality will be subject to the level of Docker and
+Kubernetes support the OS provides.
 
-Functionalities like the automatic discovery and categorization of peripherals are **not** guaranteed for macOS and Windows.
+Functionalities like the automatic discovery and categorization of peripherals
+are **not** guaranteed for macOS and Windows.
 
 
 ## Hardware Requirements
 
-In order to install the NuvlaEdge and ensure its smooth execution over time, your device should have at least:
+In order to install the NuvlaEdge and ensure its smooth execution over time,
+your device should have at least:
 
 - 512MB of RAM
 - 2GB of free disk space
 - Supported CPU architectures: AArch32, AArch64 and x86_64
 
-If you are looking at an application with constrained hardware, especially in terms of memory size, please get in touch. Some components in the [NuvlaEdge architecture](/nuvlaedge/architecture/) are optional and can be removed to save memory.  Of course it's a tradeoff.
+If you are looking at an application with constrained hardware, especially in
+terms of memory size, please get in touch. Some components in
+the [NuvlaEdge architecture](/nuvlaedge/architecture/) are optional and can be
+removed to save memory. Of course, it's a tradeoff.
 
-An remember that you should leave enough memory to host the apps that will be running on the edge device. 
+Please remember that you should leave enough memory to host the apps that will
+be running on the edge device.
 
 ## Software Requirements
 
-Before you can install the NuvlaEdge, please make sure you have:
+Before you can install the NuvlaEdge, please make sure you have Container
+Orchestration Engine (COE) installed along with the COE specific application
+deployment tools.
 
-- [Docker Engine (version 18 or higher)](https://docs.docker.com/install)
-- [Docker Compose (version 1.27.4 or higher)](https://docs.docker.com/compose/install/)
+For Docker-based edge device use:
+  - [Docker Engine (version 18 or higher)](https://docs.docker.com/engine/install/),
+  - `docker` CLI with `compose` command. Typically installed along with Docker Engine.
 
+For Kubernetes-based edge device:
```

Comment:
Around here should go the details for the kos, k3s and k8s particular configurations?